packages/@webex/plugin-cc/src/AgentConfig/types.ts

```diff
@@ -0,0 +1,54 @@
+import {AuxCode, ListTeamsResponse} from '../AgentConfigService/types';
+import {WebexSDK} from '../types';
+
+/**
+ * Represents the request to a AgentConfig
+ *
+ * @public
+ */
+export interface AgentConfigRequest {
+  /**
+   * CI user id of the User.
+   */
+  ciUserId: string;
+
+  /**
+   * Represents object of WebexSDK.
+   */
+
+  webex: WebexSDK;
+
+  /**
+   * Org id of the User.
+   */
+  orgId: string;
+}
+
+/**
+ * Represents the response from AgentConfig.
+ *
+ * @public
+ */
+export interface IAgentConfig {
```

Comment:
What about feature flags? Shouldn't we pull that data as well in the agentConfig?

Reply:
I checked the APIs I did not get anything around feature flags.